src/main/java/frc/robot/RobotManager.java

```diff
@@ -69,7 +72,9 @@ public void teleopInit() {
 
 	@Override
 	public void robotPeriodic() {
-		TimeUtils.updateCycleTime(); // Better to be first
+		roborioCycles++; // Better to be first
+		Logger.recordOutput("RoborioCycles", roborioCycles);
```

Comment:
Shouldn't this be in TimeUtils?

Reply:
it doesnt have anything to do with time

Reply:
It very much has something to do with time

Reply:
the function in `TimeUtils` is even called `updateCycleTime()`

Reply:
i personally think its robot related (like very much only general robot related), and useful in time, and that robot manager is a great place for it

Reply:
@YoniKiriaty